.github/workflows/publish-runtime-chevdor.yml

```
@@ -0,0 +1,201 @@
+        with:
+          ref: ${{ github.event.inputs.to }}
+      - name: Login to DockerHub
+        uses: docker/login-action@v2.2.0
+        if: github.repository == 'moondance-labs/tanssi'
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Download original scripts
+        uses: actions/download-artifact@v3
+        with:
+          name: original-scripts
+          path: original-scripts
+      # Build WASM with Substrate Runtime Tool
+      - name: Srtool build
+        id: srtool_build
+        uses: chevdor/srtool-actions@v0.9.2
+        with:
+          chain: ${{ matrix.chain.chain_name }}
+          runtime_dir: ${{ matrix.chain.runtime_dir }}
+      - name: Summary
+        run: |
+          echo '${{ steps.srtool_build.outputs.json }}' | jq . > ${{ matrix.chain.runtime_name }}-srtool-digest.json
+          cat ${{ matrix.chain.runtime_name }}-srtool-digest.json
+          cp ${{ steps.srtool_build.outputs.wasm_compressed }} ${{ matrix.chain.runtime_name }}-runtime.compact.compressed.wasm
+      - name: Archive Artifacts for ${{ matrix.chain }}
+        uses: actions/upload-artifact@v2
+        with:
+          name: runtime-info
+          path: |
+            ${{ matrix.chain.runtime_name }}-runtime.compact.compressed.wasm
+            ${{ matrix.chain.runtime_name }}-srtool-digest.json
+
+          
+  ####### Prepare the release draft #######
+  prepare-draft-release:
+    runs-on: ubuntu-latest
+    needs: ["setup-scripts", "build-srtool-runtimes", "matrix_prep"]
+    outputs:
+      release_url: ${{ steps.create-release.outputs.html_url }}
+      asset_upload_url: ${{ steps.create-release.outputs.upload_url }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.inputs.to }}
+          fetch-depth: 0 
+      - name: Use Node.js 18.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x  
+      - name: Download Original Tools
+        uses: actions/download-artifact@v3
+        with:
+          name: original-tools
+          path: original-tools 
+      - name: Download runtime-info
+        uses: actions/download-artifact@v3.0.2
+        with:
+          name: runtime-info
+          path: build  
+      - name: Generate release body
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        id: generate-release-body
+        working-directory: original-tools
+        run: |
+          RUNTIMES=$(echo '${{needs.matrix_prep.outputs.matrix_tests}}' | jq -r '.[] | .runtime_name + " "')
+          yarn
+          yarn -s run ts-node github/generate-runtimes-body.ts --owner "${{ github.repository_owner }}" --repo "$(basename ${{ github.repository }})" --from "${{ github.event.inputs.from }}" --to "${{ github.event.inputs.to }}" --srtool-report-folder '../build/' --runtimes $RUNTIMES > ../body.md      - name: Get runtime version
```

This looks like a typo

```suggestion
          yarn -s run ts-node github/generate-runtimes-body.ts --owner "${{ github.repository_owner }}" --repo "$(basename ${{ github.repository }})" --from "${{ github.event.inputs.from }}" --to "${{ github.event.inputs.to }}" --srtool-report-folder '../build/' --runtimes $RUNTIMES > ../body.md
```